sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/ModelsRepositoryAsyncClient.java

```diff
@@ -62,4 +73,71 @@
     public ModelsRepositoryServiceVersion getServiceVersion() {
         return this.serviceVersion;
     }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmi A well-formed DTDL model Id. For example 'dtmi:com:example:Thermostat;1'.
+     * @return A Map containing the model definition(s) where the key is the dtmi
+     * and the value is the raw model definition string.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Map<String, String>> GetModels(String dtmi) {
+        return GetModels(dtmi, this.defaultDependencyResolutionOption);
+    }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmi                       A well-formed DTDL model Id. For example 'dtmi:com:example:Thermostat;1'.
```

Comment:
Is "well-formed" adding context here? This strikes me as either worth expanding on what a well-formed DTDL model Id is vs other DTDL model Ids, or just getting rid of "well-formed" if it has no meaning

Reply:
I added a link to the specs, there is an example at the end of the comments as well.